cmd/local/conf/conf.go

```diff
@@ -354,9 +354,12 @@ func ReadConf(overrides map[string]string, configDir string) (*CollectConf, erro
 	return c, nil
 }
 
-func getOutputDir(now time.Time) string {
-	nowStr := now.Format("20060102-150405")
-	return filepath.Join(os.TempDir(), "ddc", nowStr)
+func getOutputDir(outputDir string, t time.Time) string {
+	nowStr := t.Format("20060102-150405")
+	if outputDir == "" {
+		return filepath.Join(os.TempDir(), "ddc", nowStr)
```

Comment:
need to remove call to TempDir here

Reply:
Actually thats a "safety net" if the "output dir" is empty we have to have a value in here